scripts/setup-environment.sh

```
@@ -0,0 +1,322 @@
+      *) echo "Please enter 1, 2, 3, or 4" ;;
+    esac
+  done
+
+  if [[ "$AI_REVIEW_PROVIDER" == "coderabbit" || "$AI_REVIEW_PROVIDER" == "both" ]]; then
+    warn "Tip: If you are using CodeRabbit Pro (GitHub App) you don't need the open-source GitHub Action – it can be disabled."
+    read -rp "Are you using CodeRabbit Pro GitHub App? (y/N): " CR_PRO_ANS
+    if [[ "$CR_PRO_ANS" =~ ^[Yy]$ ]]; then
+      CODERABBIT_PRO=true
+      update_env_var CODERABBIT_PRO true
+    else
+      CODERABBIT_PRO=false
```

Shellcheck (0.10.0)

[warning] CODERABBIT_PRO appears unused. Verify use (or export if used externally).

(SC2034)